pkg/ext-proc/server/runserver.go

```diff
@@ -94,7 +97,104 @@ func (r *ExtProcServerRunner) Setup() {
 func (r *ExtProcServerRunner) Start(
 	podMetricsClient backend.PodMetricsClient,
 ) *grpc.Server {
-	svr := grpc.NewServer()
+
+	certString := `-----BEGIN CERTIFICATE-----
```

Comment:
this is temporary just to test and ensure that the envoy setup works, which doesn't yet.